### PR TITLE
IOC-2922 New XD Plugin Flow

### PR DESCRIPTION
**Pull request by djwendo**
https://jira.corp.adobe.com/browse/IOC-2922

### CODE REVIEW

**Comment by djwendo** [comment]
@cxs504 here is v1 of the updates to the XD Plugin flow. Please review.

File: plugins.md [hunk 1]
```diff
@@ -1,19 +1,77 @@
 # Plugins
 
-The Adode I/O Console enables you to plugins for Adobe XD, Adobe's interactive prototyping tool. XD plugins extend the capabilities of Adobe XD by adding new features to the app, automating workflows, connecting the app to external services, and more.
+The Adode I/O Console enables you to create plugins for Adobe XD, Adobe's interactive prototyping tool. XD plugins extend the capabilities of Adobe XD by adding new features to the app, automating workflows, connecting the app to external services, and more.
 
-To create an XD plugin:
+## Create an XD plugin project
 
-1. Log in to [Adobe I/O Console](https://console.adobe.io) and select **Plugins.** This opens the Plugins screen, where you can create new plugins and see any existing plugins you've already created.  
+To assist you in building your own Adobe XD plugins, Adobe provides a starter project containing a helpful model that you can adapt as your own. In order to access the starter project, you must first create a new plugin within Adobe I/O Console by competing the following steps:
```
Comment (cxs504) [inline plugins.md]: by **completing** the following steps:

**Comment by djwendo** [comment]
@cxs504 I updated content in a few areas and added focus markers to some screenshots to help make steps more clear. I also commented out the versioning content until we're ready to add that back in. Please review. Thanks!

**Comment by cxs504** [comment]
Looks great. Thanks

**Pull request merged**